packages/bruno-cli/src/runner/interpolate-vars.js

```diff
@@ -109,7 +109,16 @@ const interpolateVars = (request, envVars = {}, collectionVariables = {}, proces
     delete request.auth;
   }
 
-  return request;
+  if (request.awsv4config) {
+    request.awsv4config.accessKeyId = _interpolate(request.awsv4config.accessKeyId) || '';
+    request.awsv4config.secretAccessKey = _interpolate(request.awsv4config.secretAccessKey) || '';
+    request.awsv4config.sessionToken = _interpolate(request.awsv4config.sessionToken) || '';
+    request.awsv4config.service = _interpolate(request.awsv4config.service) || '';
+    request.awsv4config.region = _interpolate(request.awsv4config.region) || '';
+    request.awsv4config.profileName = _interpolate(request.awsv4config.profileName) || '';
+  }
+
+  if (request) return request;
```

Comment:
This code was lifted straight from `packages/bruno-electron/src/ipc/network/interpolate-vars.js`. There is some opportunity with CLI and UI to consolidate the environment handling, but this keeps the overall PR much more surgical.